lib/cartopy/crs.py

```diff
@@ -602,8 +602,10 @@ def quick_vertices_transform(self, vertices, src_crs):
         if self == src_crs:
             x = vertices[:, 0]
             y = vertices[:, 1]
-            x_limits = self.x_limits
-            y_limits = self.y_limits
+            # Extend the limits a tiny amount to allow for precision mistakes
+            epsilon = 1.e-10
```

Comment:
I'm not sure how this is related to the intent of this PR.

Reply:
This is required to fix #403. See the very first comment in this chain where I discuss fixing two open issues with this PR.

> This request fixes an issue with plotting 10m Oceans seen in #403. The issue is that the 10m and 110m ocean shapefiles go beyond the (-180, 180) limits within the Projection quick transform by ~1.e-12, so I added in an epsilon value of 1.e-10 to allow the quick transform to proceed as long as it is close. The 50m Oceans don't have this issue.